tests/python/accuracy/public_scope.json

```diff
@@ -304,7 +304,7 @@
             {
                 "image": "coco128/images/train2017/000000000074.jpg",
                 "reference": [
-                    "anomaly_map min:0.49540239572525024 max:1.0;pred_score:1.0;pred_label:Anomalous;pred_mask min:0 max:1;"
+                    "anomaly_map min:0.495402 max:1.0;pred_score:1.0;pred_label:Anomaly;pred_mask min:0 max:1;"
```

Comment:
Can you limit the precision. We usually set 3 digits after `.`. The reason is that the exact prediction value is likely to change after new OpenVINO is released